crates/gen-host-js/src/lib.rs

```diff
@@ -17,6 +17,41 @@ use wit_bindgen_core::{
 };
 use wit_component::ComponentInterfaces;
 
+fn is_js_identifier_start(code: char) -> bool {
+    return match code as u32 {
+        0..=35 | 37..=64 | 91..=94 | 96..=96 => false,
+        36 | 65..=90 | 95 | 97..=122 => true,
```

Comment:
Could this use `'a'..='z'` style patterns instead of integer codes to be more readable? Additionally is there something online to link to as a reference here?